package.json

```diff
@@ -107,7 +107,6 @@
     "taskcluster-lib-app": "link:libraries/app",
     "taskcluster-lib-azqueue": "link:libraries/azqueue",
     "taskcluster-lib-azure": "link:libraries/azure",
-    "taskcluster-lib-azqueue": "link:libraries/azqueue",
```

Comment:
I don't think we want to remove this :)

Reply:
The same line shows up on L108.

Reply:
Oh, oops!  I guess that was a merge error!